app/views/admin/connections/index.html.slim

```diff
@@ -1,34 +1,28 @@
-.search
-  = search_form_for [:admin, search] do |f|
-    p
-      = f.label :user
-      = f.select :user_id_eq, options_for_select(User.all.map { |u| [u.email, u.id] }), include_blank: true
-    p
-      = f.label :server
-      = f.select :server_id_eq, options_for_select(Server.all.map { |s| [s.hostname, s.id] }), include_blank: true
-    p
-      = f.submit
-    = link_to "Активные", active_admin_connections_path
+= page_title('Соединения', 'bolt')
```

Comment:
I18n

Reply:
Maybe another PR plan? Lots of accommodation.

Reply:
Okay, then create a separate issue for it please